install/kubernetes/cilium/charts/config/templates/cni-configmap.yaml

```diff
@@ -27,7 +27,7 @@ data:
       "enable-debug": {{ .Values.global.cni.debug | default $defaultCniDebug }},
 {{- if .Values.global.cni.eni }}
       "eni": {
-        "first-instance-index": {{ .Values.global.cni.eni.first-interface-index | default $defaultCniFirstInterfaceIndex }}
+        "first-instance-index": {{ .Values.global.cni.eni.first_interface_index | default $defaultCniFirstInterfaceIndex }}
```

Comment:
Should be squashed with the 2nd commit.

Reply:
what do you mean? Can you please explain?